src/js/structs/PodContainer.js

```diff
@@ -30,7 +30,7 @@ module.exports = class PodContainer extends Item {
 
       default:
         return Object.assign(Object.create(PodContainerStatus.NA), {
-          displayName: StringUtil.capitalize(this.get('status').toLowerCase())
+          displayName: StringUtil.capitalize(this.getStatusString().toLowerCase())
```

Comment:
I mean we could keep `getStatusString` just move `toUpperCase` from it to the place where we actually use it